docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/deployment/.metrics-exposer.md

```diff
@@ -0,0 +1,127 @@
+# Metrics Exposer
+
+## Nebula Graph Metrics 格式
+
+```cpp
+// The raw metric format based on JSON
+// 1. Gauge, the time serial value
+// 2. Histogram, the time serial value distribution
+// The raw data can be transformed to multiple specified formats (various user-defined metric formats)
+// e.g.
+// {
+//     "name": "meta",
+//     "gauges": [...],
+//     "histograms:" [...],
+// }
+// Gauge
+// {
+//     "name": "xxxxx",
+//     "value": 33,
+//     "labels": [
+//         {"name": "name", "value": "nebula"},
+//         {"name": "type", "value": "qps"}
+//     ]
+// }
+// Histogram
+// {
+//     "name": "xxxxx",
+//     "value_range": [0, 100],
+//     "buckets": [2, 3, 0, 11, ...],
+//     "sum": 233,
+//     "count": 332,
+//     "labels": [
+//         {"name": "name", "value": "nebula"},
+//         {"name": "type", "value": "latency"}
+//     ]
+// }
+```
+
+如上述所示，**Nebula Graph** 会在各个 HTTP 服务的 `/metrics` 路径下暴露基于 JSON 的 metrics 数据，具体格式如上所述。用户可以根据需要将其转换成自己生产系统可以解读的格式，**Nebula Graph** 支持 Prometheus。如果用户使用 Prometheus，可以参考下文开始使用。
+
+## 接入 Prometheus
+
+### 接入方式
+
+Prometheus 支持 push/pull 两种 metrics 获取方式，**Nebula Graph** 支持 pull 方式，在 pull 方式下，需要 Prometheus 周期性地通过 HTTP 请求到特定的端口拉取 metrics 数据。
+
+### 启动 Nebula Graph
```

Comment:
**Nebula Graph**

Reply:
I comment out the stale doc. It will change greatly in the future.